src/main/java/mate/academy/bookstore/dto/user/UserLoginRequestDto.java

```diff
@@ -0,0 +1,16 @@
+package mate.academy.bookstore.dto.user;
+
+import jakarta.validation.constraints.Email;
+import jakarta.validation.constraints.NotBlank;
+import org.hibernate.validator.constraints.Length;
+
+public record UserLoginRequestDto(
+        @NotBlank
+        @Length(min = 8,max = 35)
+        @Email
+        String email,
+        @NotBlank
+        @Length(min = 8,max = 35)
```

Comment:
```suggestion
        @Length(min = 8, max = 35)
```
don't forget to press Ctrl Alt L regularly